qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowbridge.py

```diff
@@ -0,0 +1,68 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+import sys
+from mantidqt.widgets.helpwindow.helpwindowpresenter import HelpWindowPresenter
+from qtpy.QtCore import QUrl
+from qtpy.QtWidgets import QApplication
+
+
+_presenter = None
+
+
+def show_help_page(relative_url, local_docs=None, online_base_url="https://docs.mantidproject.org/"):
```

Comment:
Since "https://docs.mantidproject.org/" it is effectively a magic value. Pull out into the global space so it can be reused.
```py
ONLINE_BASE_URL="https://docs.mantidproject.org/"
```
then it can be used in both places